source/extensions/filters/http/ext_proc/ext_proc.cc

```diff
@@ -1673,15 +1672,17 @@ void Filter::onGrpcError(Grpc::Status::GrpcStatus status, const std::string& mes
   }
 }
 
-void Filter::onGrpcClose() {
+void Filter::onGrpcClose() { onGrpcCloseWithStatus(Grpc::Status::Aborted); }
```

Comment:
I was thinking to also expend the status to `onGrpcClose` method, but this solution is better. Thanks for adding this support.